knowledge-base/draw-chart-with-fixedcontenteditor.md

```diff
@@ -0,0 +1,162 @@
+---
+title: How to Draw a Chart with FixedContentEditor in PdfProcessing
+description: This article provides a solution to draw a bar chart with PdfProcessing. 
+type: how-to
+page_title: How to Draw a Chart with FixedContentEditor in PdfProcessing
+slug: draw-chart-with-fixedcontenteditor
+tags:  pdfprocessing, chart, draw
+res_type: kb
+---
+# Environment
+| Version | Product | Author | 
+| --- | --- | ---- | 
+| 2024.1.124 | RadPdfProcessing |[Desislava Yordanova](https://www.telerik.com/blogs/author/desislava-yordanova)| 
+
+# Description
+
+This article demonstrates a sample solution how to draw a chart in PdfProcessing.
+
+# Solution
+The powerful [FixedContentEditor]({%slug radpdfprocessing-editing-fixedcontenteditor%}) allows you draw a simulation of a chart element at the desired [Position]({%slug radpdfprocessing-concepts-position%}):
+
+ 
+   ```csharp
+            public static System.Windows.Size pageSize = new System.Windows.Size(Unit.MmToDip(210), Telerik.Windows.Documents.Media.Unit.MmToDip(297));
+        public static Padding pageMarginsValue = new Telerik.Windows.Documents.Primitives.Padding(
+             Unit.MmToDip(20),//left
+              Unit.MmToDip(0),//top
+               Unit.MmToDip(0),//right
+                Unit.MmToDip(0));//bottom
+        public static double currentYposition = pageMarginsValue.Top;
+        public static double currentXposition = pageMarginsValue.Left;
+        public static double pageYLimit = pageSize.Height - pageMarginsValue.Bottom;
+        public static double pageXLimit = pageSize.Width - pageMarginsValue.Right;
+
+        private static Random random = new Random();
+        private static Dictionary<string, int> availableLibraries = new Dictionary<string, int>();
+        private static RadFixedDocument fixedDocument = new RadFixedDocument();
+        static void Main(string[] args)
+        {
+            availableLibraries.Add("RadPdfProcessing", random.Next(20, 500));
+            availableLibraries.Add("RadSpreadProcessing", random.Next(20, 500));
+            availableLibraries.Add("RadWordsProcessing", random.Next(20, 500));
+            availableLibraries.Add("RadSpreadStreamProcessing", random.Next(20, 500));
+            availableLibraries.Add("RadZipLibrary", random.Next(20, 500)); 
+
+            DrawChart();
+
+            PdfFormatProvider provider = new PdfFormatProvider();
+            string outputFilePath = @"..\..\sample.pdf";
+            File.Delete(outputFilePath);
+            using (Stream output = File.OpenWrite(outputFilePath))
+            {
+                provider.Export(fixedDocument, output);
+            }
+            Process.Start(outputFilePath);
```

Comment:
This might not work in every environment, Maybe use 
` 
var psi = new ProcessStartInfo()
{
FileName = saveFile,
 UseShellExecute = true
 };
Process.Start(psi);
`